.codecov.yml

```diff
@@ -10,7 +10,7 @@ coverage:
   status:
     project:
       default:
-        target: 81%
+        target: 80%
```

Comment:
Trying comments for issue #69

Reply:
Very good. It even shows up as a "resolvable" comment